docs/particles/particle_class.rst

```diff
@@ -1,25 +1,25 @@
 .. _particle-class:
 
-`~plasmapy.particles.particle_class.Particle` Class
-***************************************************
+|Particle| Class
+****************
```

Comment:
I'm surprised substitutions in headings don't produce build warnings due to the length of underline being shorter than the heading after the substitution.  I'm really curious what's going on behind the scenes here, but I'm going to resist going down the rabbit hole.

Reply:
Given my experience with rabbit holes, we should probably rename them into dragon holes.